src/views/Simulation/components/Kpis/KpiCard.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card, Grid, Typography } from '@mui/material';
+import KpiTrendChip from './KpiTrendChip';
+
+const KpiCard = ({ kpi }) => {
+  return (
+    <Card sx={{ width: '100%', p: 2 }}>
+      <Grid container justifyContent="space-between" alignItems="center" columnSpacing={2}>
+        <Grid item xs="auto">
```

Comment:
`xs="auto"` is the same thing as `xs` ?

Reply:
Not exactly, according to [the documentation](https://v5.mui.com/material-ui/react-grid/#variable-width-content):
> Set one of the size breakpoint props to "auto" instead of true / a number to size a column based on the natural width of its content.